Source/Raymarcher/Shaders/Private/WindowedSampling.usf

```diff
@@ -50,3 +50,33 @@ float4 SampleWindowedVolumeOctreeStep(int3 CurPos, float StepSize, Texture3D Vol
 	const float DataValue = Volume.Load(MipLevelPos, 0).r;
 	return SampleWindowedTransferFunction(DataValue, StepSize, TF, TFSampler, WindowingParams);
 }
+
+// Returns a bit number where all the bits between the window center and window width are ones.
+uint GetWindowingParamsBitNumber(float WindowCenter, float WindowWidth)
+{
+	// TFPos == 1 => Value = WindowWidth - WindowWidth/2 + WindowCenter
+	// TFPos == 0 => Value = WindowCenter - (WindowWidth/2);
+    
+	float Value0 = WindowCenter - (WindowWidth / 2.0);
+	float Value1 = WindowWidth / 2.0 + WindowCenter;
```

Comment:
Please turn the operators around and add a parenthesis, so the two lines are consistent 😂